extensions/2.0/Khronos/KHR_texture_video/KHR_texture_video.md

```diff
@@ -0,0 +1,155 @@
+# KHR_texture_video
+
+## Contributors
+
+* Dwight Rodgers, Adobe
+* Gary Hsu, Microsoft
+* Emmett Lalish, Google
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension allows glTF models to use videos as textures. A client that does not implement this extension can ignore the provided video and continue to rely on the PNG and JPG textures available in the base specification. Defining a fallback texture is optional. 
+
+Video textures support use cases such as water or other textures that continuously repeat, coordination of video texture playback by animation (in conjunction with `KHR_animation_pointer`), or control of video texture playback by interactivity (in conjunction with `KHR_interactivity`).
+
+## glTF Schema Updates
+
+The `KHR_texture_video` extension is added to the `textures` node and specifies a `source` property that points to the index of the `images` node which in turn points to the video.
+
+The following glTF will load `texture.webm` in clients that support this extension, and otherwise fall back to `image.png`.
+
+```
+"textures": [
+    {
+        "source": 0,
+        "extensions": {
+            "KHR_texture_video": {
+                "source": 0,
+                "playhead": 0,
+                "autoplay": true,
+                "loop": true
+            }
+        }
+    }
+],
+"extensions": {
+    "KHR_texture_video": {
+        "videos": [
+            "uri": "texture.webm"
+        ]
```

Comment:
This is not valid JSON, the `[]` symbols specify an array but you are defining a property like a sub-JSON.

Reply:
Thanks for catching this. I believe it is resolved in the latest commit. Please let me know if you see any further problems with this.